core/server/master/src/test/java/alluxio/master/AlluxioMasterProcessTest.java

```diff
@@ -209,17 +248,33 @@ public void restoreFromBackupLocal() throws Exception {
   }
 
   private void startStopTest(AlluxioMasterProcess master) throws Exception {
+    startStopTest(master, true, true);
+  }
+
+  private void startStopTest(AlluxioMasterProcess master,
```

Comment:
Thanks for the refactor. Looks a lot better now.

Reply:
😄, Thanks for your suggestion!